build_scripts/publish_and_deploy.sh

```diff
@@ -0,0 +1,5 @@
+docker push $1/todo-app:latest
```

Comment:
Minor, but if we moved these first couple of lines into e.g. a before_deploy stage, then we'd be able to refer to the variables with names rather than just argument positions